codespell_lib/data/dictionary.txt

```diff
@@ -21089,23 +21101,53 @@ keyworkds->keywords
 keyworks->keywords, key works,
 keywors->keywords
 keywprd->keyword
+kibutz->kibbutz
+kibutzes->kibbutzim
+kibutzim->kibbutzim
+kidknap->kidnap
+kidknapped->kidnapped
+kidknappee->kidnappee
+kidknappees->kidnappees
+kidknapper->kidnapper
+kidknappers->kidnappers
+kidknapping->kidnapping
+kidknaps->kidnaps
+kighbosh->kibosh
+kighboshed->kiboshed
+kighboshes->kiboshes
+kighboshing->kiboshing
+kimerra->chimera
```

Comment:
Add `kimera`?

And `kimerical`/`kimerrical`?

Reply:
> Add `kimera`?

Added.

> And `kimerical`/`kimerrical`?

Added. Also some others as per [Merriam-Webster](https://www.merriam-webster.com/dictionary/chimerical).